src/app/Plans/components/Wizard/FilterVMsForm.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { Form, FormGroup, TreeView, Title } from '@patternfly/react-core';
+
+// TODO replace with redux or equiv.
+import { VMsOptions } from './mocks/VMs.mock';
+
+interface IFilterVMsProps {
+  Inventory: [];
+}
+
+const FilterVMs: React.FunctionComponent<IFilterVMsProps> = ({ Inventory }: IFilterVMsProps) => {
+  const [activeItems, setItems] = React.useState(VMsOptions);
+  const [checkedItems, setCheckedItems] = React.useState([]);
+
+  const onSelect = (event, treeViewItem, parentItem) => {
+    return;
+  };
+
+  const onCheck = (event, treeViewItem) => {
+    const checked = event.target.checked;
+    treeViewItem.checkProps.checked = checked;
```

Comment:
@gildub , sorry if my earlier comment got lost in the mess, can you remove this `checked` assignment here and just have `onCheck` be a no-op `return;` like `onSelect` for now? Mutating state in-place like this is causing weird side-effects when trying to use the checkboxes, we should just have them totally do nothing until we implement #38.

Reply:
@mturley, no worries, done.